File: go/1005/main.go
package main

import ("fmt")

func averageOne() string {
	var a, b float64
	weightOfA := 3.5
	weightOfB := 7.5

	// Lê os valores de entrada
	fmt.Scan(&a)
	fmt.Scan(&b)

	// Calcula a média ponderada
	media := (a*weightOfA + b*weightOfB) / (weightOfA + weightOfB)

	// Retorna o resultado formatado
	return fmt.Sprintf("MEDIA = %.5f", media)
}

func main() {
	fmt.Println(averageOne())
}

File: php/1005/main.php
<?php

function averageOne()
{
    // Lê os valores de entrada
    $a = (float)trim(fgets(STDIN));
    $b = (float)trim(fgets(STDIN));

    // Define os pesos
    $weightOfA = 3.5;
    $weightOfB = 7.5;

    // Calcula a média ponderada
    $media = ($a * $weightOfA + $b * $weightOfB) / ($weightOfA + $weightOfB);

    // Retorna o resultado formatado
    return sprintf("MEDIA = %.5f", $media);
}

echo averageOne() . PHP_EOL;
